### ISSUE TITLE
http://buchen.github.io/maven-repo is not available

### ISSUE THREAD

**Issue by alibabajdk** [buchen/fix-info-plist-maven-plugin#3]
http://buchen.github.io/maven-repo is not available
It can not be reached

**Comment by buchen** [comment]
Does Maven tell you this?

The metadata files and all are there. But there are no "index.html" for the brewer to display.
http://buchen.github.io/maven-repo/name/abuchen/fix-info-plist-maven-plugin/maven-metadata.xml

**Comment by cykl** [comment]
Hi @buchen, 

Note: I am hijacking this issue but it might be related.

I just discover this project while building [OpenJDK JMC](http://openjdk.java.net/projects/jmc/7/) which depend on it. Having your own repo means that people may have to configure proxy setting, when possible, or to ask someone to configure their internal Nexus to proxy your repo. Our life could be much easier if you uploaded to a well known repository. Is there a reason to no use one of them?

**Comment by buchen** [comment]
Hi @cykl,

to be honest, I did not know of any significant usage of this plugin besides by [my own RCP project](https://github.com/buchen/portfolio) and never bothered to publish it somewhere else.

Is Maven Central the way to go?

**Comment by buchen** [comment]
@cykl - I have now published version 1.5 to Maven Central. You should be able to use it without the custom repository.

I have also added the option to add native NSObjects so that ```UTExportedTypeDeclarations``` and ```CFBundleDocumentTypes``` could be implemented.

For example:

```xml
<plugin>
  <groupId>name.abuchen</groupId>
  <artifactId>fix-info-plist-maven-plugin</artifactId>
  <version>1.5</version>
  ...
  <configuration>
    <properties>
      <property>
        <name>CFBundleDocumentTypes</name>
        <value><![CDATA[<array><dict><key>CFBundleTypeIconFiles</key><array><string>rip-22x29.png</string></array></dict></array>]]</value>
    </properties>
  </configuration>
</plugin>
```

**Comment by cykl** [comment]
:+1: 

I will notify jmc team that your artifact is now on Maven central. Thanks!
